x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/parallel.playwright.config.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createPlaywrightConfig } from '@kbn/scout';
+
+// eslint-disable-next-line import/no-default-export
+export default createPlaywrightConfig({
+  testDir: './tests/',
```

Comment:
for consistency with other plugins let's put these tests under `./parallel_tests`

Sequentially running tests will be located in `./tests`